run-a-node.md

```diff
@@ -168,8 +168,9 @@ After=network.target
 [Service]
 Type=simple
 User=kuji
-ExecStart=/home/kuji/go/bin/kujirad start --log-level error 
```

Comment:
@checkso is the removal of `log-level` intentional?

Reply:
someone mentioned it is better without it, as you can see all messages. 

If we want to keep it, it must be **--log_level error** 
_ not -